docs/intro.rst

```diff
@@ -418,7 +466,15 @@ in this table; you can find them in section
 |                          +--------------------+----------------------+-------+
 |                          | swig               | >=2.0                | py2   |
 +--------------------------+--------------------+----------------------+-------+
-| Windows                  | None               |                      |       |
+| Windows (native)         | None               |                      |       |
++--------------------------+--------------------+----------------------+-------+
+| Windows (UNIX-like env)  | OpenSSL developm.  |                      | py2   |
```

Comment:
The name is openssl-devel

Reply:
DONE. I updated all packages and rescoped that group of table rows to only CygWin.